src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/ExceptionRecordCreationTest.java

```diff
@@ -33,18 +36,25 @@
     @Autowired
     private EnvelopeMessager envelopeMessager;
 
+    @Autowired
+    DocumentManagementUploadService dmUploadService;
+
     @DisplayName("Should create ExceptionRecord when provided/requested supplementary evidence is not present")
     @Test
     public void create_exception_record_from_supplementary_evidence()
         throws JSONException, InterruptedException, ServiceBusException {
         // given
         UUID randomPoBox = UUID.randomUUID();
-
+        UploadResponse uploadResponse = dmUploadService.uploadToDmStore(
+            "supplementary-evidence.pdf",
+            "documents/supplementary-evidence.pdf"
+        );
         // when
         envelopeMessager.sendMessageFromFile(
             "envelopes/supplementary-evidence-envelope.json",
             "0000000000000000",
-            randomPoBox
+            randomPoBox,
+            ScannedDocumentsHelper.getScannedDocumentUrl(uploadResponse)
```

Comment:
should we not check that the exception record has the url on it on the `//then` stage?

Reply:
This test is about the Exception Record. So I think we dont need to check the dm url here

Reply:
we may as well check that the document is now there, won't hurt